client/src/pages/Courses/ClassCard.tsx

```diff
@@ -48,34 +50,62 @@ const useStyles = makeStyles({
   },
 });
 
-const ClassCard: React.FC<ClassProps> = (props) => {
+const ClassCard: React.FC<ClassCardData> = ({
+  id, 
+  name,
+  code,
+  educator,
+  description,
+  uid,
+  refresh
+}) => {
+  const { deleteCourseContext, userData } = useContext(UserContext);
   const history = useHistory();
   const classes = useStyles();
-  const loadCourse = (id: string) => {
-    console.log(props.id);
+  const loadCourse = () => {
     history.push({
       pathname: "/coursepage",
       search: id,
     });
   };
 
+  const unenroll = async () => {
+    const res = await axios.post("api/courses/unenroll", {
```

Comment:
Did this not work when you imported the function from services?

Reply:
+1 to move the function to services for consistency / keeping all axios calls in one place